src/model/scripts/bpe_training/sp_token.json

```diff
@@ -0,0 +1,9 @@
+{
+    "special_tokens": [
+        "<s>",
+        "<pad>",
+        "</s>",
+        "<unk>",
+        "<mask>"
```

Comment:
Do we need
<s> 
</s>
<mask>
token for autoregressive model tokenizsr

Reply:
now i have only <eos> and <unk>